### ISSUE TITLE
no space between caracter in select box

### ISSUE THREAD

**Issue by marius851000** [EddyK28/PMD-Lua-Mod-Loader#1]
Open a dialog box ( eg : pokemorph ).
There is no space in "SelectPokemon" and "ForceLevel".

In super mystery dungeon, with the latest version of the depo.

**Comment by EddyK28** [comment]
Hmm, that's odd, I can't seem to replicate this.  What language is your game running?

**Comment by marius851000** [comment]
I'm running in french ( E.U. version )

**Comment by marius851000** [comment]
Still have this issue with E.U. / en

**Comment by EddyK28** [comment]
Interesting.  It looks like the text directive that I've used for spaces is interpreted differently by the EU version of the game.  I've switched it out for a directive that should be guaranteed to represent space, and tested on an EU game with positive results.

Just to be certain, please run the following code snippet and tell me which sets of parenthesis contain a space when printed.
```lua
WINDOW:SysMsg("( )--([LE])--(쬀)--([S:3])--(쐃)")
WINDOW:CloseMessage()
```

**Comment by EddyK28** [comment]
Should be fixed by ffba7e2. Closing this for now.
